src/lib/parserRaw.c

```diff
@@ -572,6 +572,97 @@ RdbStatus elementRawSet(RdbParser *p) {
     }
 }
 
+RdbStatus elementRawZsetLP(RdbParser *p) {
+    return singleStringTypeHandling(p, listpackValidateIntegrityCb, "elementRawZsetLP");
+}
+
+RdbStatus elementRawZsetZL(RdbParser *p) {
+    return singleStringTypeHandling(p, ziplistValidateIntegrityCb, "elementRawZsetZL");
+}
+
+RdbStatus elementRawZset(RdbParser *p) {
+    enum RAW_ZSET_STATES {
+        ST_RAW_ZSET_HEADER=0,
+        ST_RAW_ZSET_READ_MEMBER_STR,
+        ST_RAW_ZSET_READ_MEMBER_STR_RETURN,
+        ST_RAW_ZSET_READ_SCORE_DOUBLE,
+        ST_RAW_ZSET_READ_SCORE_STR,
+        ST_RAW_ZSET_READ_SCORE_STR_RETURN,
+    };
```

Comment:
this is probably the best place to explain the state-flow , next to each state. For reference see comments at  `enum RAW_LIST_STATES`.

Reply:
done